bundle/README.md

```diff
@@ -0,0 +1,31 @@
+# CRI-O static build bundle
+
+The following runtime dependencies are needed to let CRI-O work in conjunction
+with this bundle:
+
+- [runc][0]
+- [conmon][1]
+- [CNI plugins][2]
+
+[0]: https://github.com/opencontainers/runc
+[1]: https://github.com/containers/conmon
+[2]: https://github.com/containernetworking/plugins
+
```

Comment:
does/will crun come into play here at some point?

Reply:
Hm yeah I just wanted to stick to the minimal requirements. :)